query/templates/search/v710/contentQuery.tmpl

```diff
@@ -6,14 +6,18 @@
  "size" : {{.Size}},
  "query" : {
      "bool" : {
-       "must" : {
+          {{- if .Topic}}
```

Comment:
This if else statement, does this mean if there is content type filter and a topics filter then it will only filter by topic?